### PR TITLE
[LRA-11] - Notification component

### PR DESCRIPTION
**Pull request by KonstantinChabanenko**
- antd styles were imported into App.css
- Were created sagas for success and error notifications
- Was used notification component from antd
- Notifications content was placed in separate file
- Interfaces for notifications objects were created

### CODE REVIEW

File: src/store/sagas/notificationsSaga.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { takeEvery } from 'redux-saga/effects';
+import { productActionTypes } from '../actionTypes/productActionTypes';
+import { notification } from 'antd';
+import notifications from '../../resources/notifications';
+
+function* successProductCreation() {
+  yield notification.success(notifications.success);
+}
```
Comment (mkozachok) [inline src/store/sagas/notificationsSaga.ts]: `function* successProductCreation(action) {
  yield notification.success(action.message);
}`

the message text should be passed from action

File: src/components/forms/UpdateProductForm/UpdateProductForm.tsx [hunk 2]
```diff
@@ -44,7 +44,7 @@ export const UpdateProductForm = (props: TParams) => {
         validationSchema={AddProductSchema}
         render={ProductForm}
         onSubmit={(values, actions) => {
-          productAddSuccess();
+          dispatch({ type: productActionTypes.UPDATE_PRODUCT__START });
```
Comment (KonstantinChabanenko) [inline src/components/forms/UpdateProductForm/UpdateProductForm.tsx:47]: temporary stub

File: src/store/sagas/notificationsSaga.ts [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import { takeEvery } from 'redux-saga/effects';
```
Comment (mkozachok) [inline src/store/sagas/notificationsSaga.ts]: maybe rename filename to productNotificationSagas.ts

File: src/store/sagas/notificationsSaga.ts [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+function* successProductCreation() {
+  yield notification.success(notifications.productCreationSuccess);
+}
+
+function* failureProductCreation() {
+  yield notification.error(notifications.productCreationError);
+}
+
+function* successProductDeletion() {
+  console.log("The product was deleted");
```
Comment (mkozachok) [inline src/store/sagas/notificationsSaga.ts]: remove console.log()

File: src/store/sagas/notificationsSaga.ts [hunk 5]
```diff
@@ -0,0 +1,38 @@
+import { takeEvery } from 'redux-saga/effects';
+import { productActionTypes } from '../actionTypes/productActionTypes';
+import { notification } from 'antd';
+import notifications from '../../resources/notifications';
+
+function* successProductCreation() {
```
Comment (mkozachok) [inline src/store/sagas/notificationsSaga.ts]: should end with `saga`

File: src/resources/notifications.ts [hunk 6]
```diff
@@ -0,0 +1,30 @@
+import { INotifications } from "../types/notifications";
+
+const notifications: INotifications = {
```
Comment (mkozachok) [inline src/resources/notifications.ts]: think about separate by entity type - `productNotifications`, `orderNotification`, etc

**Review by mkozachok — CHANGES_REQUESTED** [review]

**Review by mkozachok — APPROVED** [review]

**Pull request merged**